src/benchmarks.jl

```diff
@@ -11,8 +11,8 @@ function timer(f::Function, n::Integer)
 end
 
 function benchmark(f::Function,
-                   category::String,
-                   name::String,
+                   category::AbstractString,
```

Comment:
Don't tag this without either adding `julia 0.4` to `REQUIRE`, or depending on Compat.jl.